insertion_sort/InsertionSort.cs

```diff
@@ -0,0 +1,31 @@
+using static System.;
```

Comment:
Don't use static, use `using System;`.

Reply:
@mohitkyadav it was in the Sample code. Should i remove this ??

Reply:
Yes @Lucifer24 please proceed. Using `static` in code will lead to travis build failure.